asset-manager.php

```diff
@@ -59,34 +60,36 @@ function am_enqueue_script( $handle, $src = false, $deps = array(), $condition =
 	 * Modify the load method of an already-enqueued script
 	 *
 	 * @param string $handle      Handle for script.
-	 * @param string $load_method How to load this asset
+	 * @param string $load_method How to load this asset.
 	 */
 	function am_modify_load_method( $handle, $load_method = 'sync' ) {
 		Asset_Manager_Scripts::instance()->modify_load_method( $handle, $load_method );
 	}
 
 endif;
 
-add_action( 'after_setup_theme', array( 'Asset_Manager_Scripts', 'instance' ), 10 );
+add_action( 'after_setup_theme', [ 'Asset_Manager_Scripts', 'instance' ], 10 );
 
 if ( ! function_exists( 'am_enqueue_style' ) ) :
 
 	/**
 	 * Load an external stylesheet. Options can be passed in as an array or individual parameters.
 	 *
-	 * @param string $handle 	  Handle for stylesheet. This is necessary for dependency management.
-	 * @param string $src 		  URI to stylesheet.
+	 * @param string $handle      Handle for stylesheet. This is necessary for dependency management.
+	 * @param string $src         URI to stylesheet.
+	 * @param array  $deps        List of dependencies.
 	 * @param string $condition   Corresponds to a configured loading condition that, if matches, will allow the stylesheet to load. Defaults are 'global', 'single', and 'search'.
 	 * @param string $load_method How to load this asset.
-	 * @param string $load_hook   Hook on which to load this asset
-	 * @param string $media 	  Media query to restrict when this asset is loaded
+	 * @param string $version      Version of the script.
```

Comment:
Should the CS have caught this, or is the alignment not required?

```suggestion
	 * @param string $version     Version of the script.
```

Reply:
Interesting. It looks like the WordPress standards explicitly don't include this sniff. TIL

```
<!-- Excluded to allow param documentation for arrays  -->
<exclude name="Squiz.Commenting.FunctionComment.SpacingAfterParamName"/>
```